src/OrbitGl/TranslationStack.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRANSFORMATION_STACK_H_
+#define ORBIT_GL_TRANSFORMATION_STACK_H_
+
+#include <vector>
+
+#include "CoreMath.h"
+
+namespace orbit_gl {
+class TranslationStack {
```

Comment:
This class would benefit from a unit test since it's now factored out. ;-)

Reply:
Huh. Actually I wrote unit tests for this, where did they go? Let me go on a quest to find the lost tests...

Reply:
Ah, right, it's already tested implicitly by the Batcher tests, that's why I didn't write explicit test cases. Let me add some explicit tests here, I'll keep the ones in `Batcher` anyways because they test at a slightly higher level.

Reply:
Thanks for the tests!